pkg/catalogv2/content/content_test.go

```diff
@@ -193,7 +193,199 @@ func TestFilterReleases(t *testing.T) {
 			}
 			contentManager := Manager{}
 			settings.ServerVersion.Set(tt.rancherVersion)
-			contentManager.filterReleases(&filteredIndexFile, nil)
+			contentManager.filterReleases(&filteredIndexFile, nil, false)
+			result := reflect.DeepEqual(indexFile, filteredIndexFile)
+			assert.Equal(t, tt.expectedPass, result)
+			if result != tt.expectedPass {
+				t.Logf("Expected %v, got %v for %s with rancher version %s", tt.expectedPass, result, tt.chartVersionAnnotation, tt.rancherVersion)
+			}
+		})
+	}
+}
+
+func TestNotFilteringReleases(t *testing.T) {
+	tests := []struct {
+		testName               string
+		chartVersionAnnotation string
+		rancherVersion         string
+		expectedPass           bool
+	}{
+		{
+			"rancher version in range comparison with `>= <`style comparison",
+			">= 2.5.0-alpha3 <2.6.0",
+			"v2.5.0+123", //SemVer comparisons using constraints without a prerelease comparator will skip prerelease versions
+			true,
+		},
```

Comment:
nit: this is more of a semver test than a functionality test. Since we have all those other semver tests I think you could just have a true/false test here and be good. But otherwise good job adding tests.